tools/ubuntu-setup/docker.sh

```diff
@@ -11,16 +11,17 @@ sudo apt-get -y update -qq
 sudo apt-get -y install linux-image-extra-$(uname -r)
 
 # DOCKER
-sudo apt-get install -q -y --force-yes lxc-docker-1.9.0
+sudo apt-get install -q -y --force-yes lxc-docker-1.10.3
```

Comment:
does this work? when I made 1.10 experiments I had to change the apt source to 

apt-get install apt-transport-https ca-certificates

sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D

deb https://apt.dockerproject.org/repo ubuntu-trusty main